### ISSUE TITLE
Android Device buttons

### ISSUE THREAD

**Issue by cnguyen2049** [python490/Vozie#5]
On my device which is has a smaller screen size than most devices. The Yes and No buttons appear right on the search bar.

**Comment by python490** [comment]
For some reason on this branch, I get a GoogleAPI authentication error. I even tried to change the API key to my own.

**Comment by cppshane** [comment]
Yeah I have the biggest phone on the market (Nexus 6) so I haven't tested it on a smaller device. Definitely will check this out.

**Comment by cnguyen2049** [comment]
Yeah I will send a screenshot when I can this week. I will be moving onto a
bigger phone as well in the coming months so hopefully I can still test on
my smaller device in the future.
